### ISSUE TITLE
Empatica Electodermal Activity or Biovotion Galvanic Skin Response

### ISSUE THREAD

**Issue by afolarin** [RADAR-base/RADAR-Schemas#80]
Unless I'm mistaken these are the same thing from the sensor perspective. Should probably harmonise on a single definition.

see:
http://qsensor-support.affectiva.com/customer/en/portal/articles/1539261-is-eda-the-same-as-gsr-

https://github.com/RADAR-CNS/RADAR-Schemas/blob/master/commons/passive/biovotion/biovotion_vsm1_galvanic_skin_response.avsc

https://github.com/RADAR-CNS/RADAR-Schemas/blob/master/commons/passive/empatica/empatica_e4_electrodermal_activity.avsc

**Comment by blootsvoets** [comment]
Indeed, the empatica docs state that EDA is measured with a GSR sensor. E4 unit is microsiemens, the reciprocal of kOhm used in Biovotion. We’d have to look at what unit conversion would do to precision.
